src/binding.cc

```diff
@@ -995,156 +972,120 @@ class Client : public ObjectWrap {
-      close_symbol = NODE_PSYMBOL("close");
-      insert_id_symbol = NODE_PSYMBOL("insertId");
-      affected_rows_symbol = NODE_PSYMBOL("affectedRows");
-      num_rows_symbol = NODE_PSYMBOL("numRows");
-      code_symbol = NODE_PSYMBOL("code");
-      cfg_user_symbol = NODE_PSYMBOL("user");
-      cfg_pwd_symbol = NODE_PSYMBOL("password");
-      cfg_host_symbol = NODE_PSYMBOL("host");
-      cfg_port_symbol = NODE_PSYMBOL("port");
-      cfg_socket_symbol = NODE_PSYMBOL("unixSocket");
-      cfg_db_symbol = NODE_PSYMBOL("db");
-      cfg_timeout_symbol = NODE_PSYMBOL("connTimeout");
-      cfg_secauth_symbol = NODE_PSYMBOL("secureAuth");
-      cfg_multi_symbol = NODE_PSYMBOL("multiStatements");
-      cfg_compress_symbol = NODE_PSYMBOL("compress");
-      cfg_metadata_symbol = NODE_PSYMBOL("metadata");
-      cfg_ssl_symbol = NODE_PSYMBOL("ssl");
-      cfg_ssl_key_symbol = NODE_PSYMBOL("key");
-      cfg_ssl_cert_symbol = NODE_PSYMBOL("cert");
-      cfg_ssl_ca_symbol = NODE_PSYMBOL("ca");
-      cfg_ssl_capath_symbol = NODE_PSYMBOL("capath");
-      cfg_ssl_cipher_symbol = NODE_PSYMBOL("cipher");
-      cfg_ssl_reject_symbol = NODE_PSYMBOL("rejectUnauthorized");
-      cfg_local_infile_symbol = NODE_PSYMBOL("local_infile");
-      cfg_read_default_file = NODE_PSYMBOL("read_default_file");
-      cfg_read_default_group = NODE_PSYMBOL("read_default_group");
-      cfg_charset_symbol = NODE_PSYMBOL("charset");
-
-      target->Set(name, Client_constructor->GetFunction());
+      NanScope();
+
+      Local<String> name = NanNew<String>("Client");
+
+      Local<FunctionTemplate> tpl = NanNew<FunctionTemplate>(New);
+      tpl->InstanceTemplate()->SetInternalFieldCount(1);
+      tpl->SetClassName(name);
+
+      NanAssignPersistent(Client_constructor, tpl);
+
+      NODE_SET_PROTOTYPE_METHOD(tpl, "connect", Connect);
+      NODE_SET_PROTOTYPE_METHOD(tpl, "query", Query);
+      NODE_SET_PROTOTYPE_METHOD(tpl, "abortQuery", AbortQuery);
+      NODE_SET_PROTOTYPE_METHOD(tpl, "escape", Escape);
+      NODE_SET_PROTOTYPE_METHOD(tpl, "end", Close);
+      NODE_SET_PROTOTYPE_METHOD(tpl, "isMariaDB", IsMariaDB);
+
+      // Make it visible in JavaScript land
+      target->Set(name, tpl->GetFunction());
     }
 };
 
-static Handle<Value> Escape(const Arguments& args) {
-  HandleScope scope;
+static NAN_METHOD(Escape) {
+  NanScope();
 
   if (args.Length() == 0 || !args[0]->IsString()) {
-    return ThrowException(Exception::Error(
-      String::New("You must supply a string"))
-    );
+    return NanThrowError("You must supply a string");
   }
+
   String::Utf8Value arg_v(args[0]);
   unsigned long arg_len = arg_v.length();
   char *result = (char*) malloc(arg_len * 2 + 1);
-  unsigned long result_len =
-          mysql_escape_string_ex(result, (char*)*arg_v, arg_len, "utf8");
-  Local<String> escaped_s = String::New(result, result_len);
+  unsigned long result_len = mysql_escape_string_ex(result, (char*)*arg_v, arg_len, "utf8");
```

Comment:
This was previously wrapped to a new line because the line exceeds 80 characters.